### ISSUE TITLE
Phrase editor

### ISSUE THREAD

**Issue by grammoboy2** [sjaehn/BJumblr#20]
When working with it, you start to think about the option you'll find in non-sequencer. Most notably the phrase editor, where you can arrange and play the made patterns in a certain sequence (to make a song / beat).

But in this case you probably want just one pattern play at once. 

Hm maybe a combination of the Phrase Editor and the Pattern Triggers. Some way to be able to sequence patterns. With the ability to repeat a certain pattern multiple times. 4x pattern 1, then 4x pattern 4 .... 8x pattern 2... 

http://non.tuxfamily.org/sequencer/doc/MANUAL.html#n:2.2.

http://non.tuxfamily.org/sequencer/doc/non-phrase-editor.png
http://non.tuxfamily.org/sequencer/doc/non-pattern-triggers.png

**Comment by sjaehn** [comment]
Good idea. But it will take some time to implement. In the meantime you can combine B.Jumblr with a sequencer and use its MIDI signals to control the patterns.
